ui/mirage/config.js

```diff
@@ -932,18 +932,32 @@ export default function () {
   });
 
   this.post('/sentinel/policy/:id', function (schema, req) {
-    const { Name, Description, Rules } = JSON.parse(req.requestBody);
+    const { Name, Description, EnforcementLevel, Policy, Scope } = JSON.parse(
+      req.requestBody
+    );
     return server.create('sentinelPolicy', {
       name: Name,
       description: Description,
-      rules: Rules,
+      enforcementLevel: EnforcementLevel,
+      policy: Policy,
+      scope: Scope,
```

Comment:
Before, mirage was only equipped to handle a trite implementation of our POST sentinel policies endpoint. Now it's slightly less trite.